packages/dapi/lib/grpcServer/handlers/createGrpcErrorFromDriveResponse.js

```
@@ -64,6 +64,9 @@
   const message = decodedInfo.message;
   const data = decodedInfo.data || {};
 
+  const serializedConsensusError = data.serializedError;
+  delete data.serializedError;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)